app/pages/feed/feed.html

```diff
@@ -2,6 +2,9 @@
 
 <ion-content padding class="feed">
 
+    <user-status-card [user]="authUser.userInfo"
+                      [onClickPage]="userStatusClickPage"></user-status-card>
```

Comment:
Adds the new user status card to the top of the news feed page.

The card takes in as arguments the user and the destination page that clicking on the card navigates to.